packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -33,43 +33,48 @@
                 ->class(['fi-fo-textarea overflow-hidden'])
         "
     >
-        <textarea
-            x-ignore
-            @if (FilamentView::hasSpaMode())
-                ax-load="visible"
-            @else
-                ax-load
-            @endif
-            ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
-            x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
-            @if ($shouldAutosize)
-                x-intersect.once="resize()"
-                x-on:input="resize()"
-                x-on:resize.window="resize()"
-            @endif
-            x-bind:style="{ height }"
```

Comment:
Now this has been removed, are we actually using the height property anymore? Can it be removed?